assets/html/volunteer.html

```diff
@@ -0,0 +1,373 @@
+          rel="noopener noref">guide from
+          Code for America</a>.
+      </p>
+      <p>
+        Code for San José is dedicated to providing a <span>
+          harassment-free
+          experience for everyone
+        </span> regardless of <span>
+          gender, gender identity and
+          expression, sexual orientation, disability, physical appearance, body
+          size, race, age, or religion
+        </span>. <span>
+          We do not tolerate harassment of staff,
+          presenters, and participants in any form
+        </span>. <span>
+          Sexual language and imagery
+          is not appropriate
+        </span> for any Code for America event or network activity,
+        including talks. <span>Anyone in violation of these policies may be expelled</span>
+        from Code for America network activities, events, and digital forums,
+        at the discretion of the event organizer or forum administrator.
+      </p>
+      <p>
+        <span>Harassment includes but is not limited to:</span>
+        <ul>
+          <li>Offensive verbal or written
+            comments related to:</li>
+          <ul>
+            <li>Gender</li>
+            <li>Gender identity and expression</li>
+            <li>Sexual orientation</li>
+            <li>Disability</li>
+            <li>Physical appearance</li>
+            <li>Body size</li>
+            <li>Race</li>
+            <li>Religion</li>
+          </ul>
+          <li>Sexual images in public spaces</li>
+          <li>Deliberate intimidation</li>
+          <li>Stalking</li>
+          <li>Following</li>
+          <li>Harassing photography or recording</li>
+          <li>Sustained disruption of talks or other events</li>
+          <li>Inappropriate physical contact</li>
+          <li>Unwelcome sexual attention</li>
+          <li>Unwarranted exclusion</li>
+          <li>Patronizing language or action</li>
+        </ul>
+      </p>
+      <p>
+        If a participant engages in harassing behavior, the organizers may
+        take any action they deem appropriate, including warning the offender
+        or expulsion from Code for San José network activities, events, and
+        digital forums.
+      </p>
+      <p>
+        If you are being harassed, notice that someone else is being harassed,
+        or have any other concerns, please contact Code for San José Captains
+        <a href="mailto:codeforsanjose@gmail.com" target="_blank" rel="noopener noref">Annie Steenson, Joseph Richardson
+          and Yan-Yin Choy</a>.
+      </p>
+      <p>
+        Event staff or forum administrators will be happy to help participants
+        contact hotel/venue security or local law enforcement, provide
+        escorts, or otherwise assist those experiencing harassment to feel
+        safe for the duration of the event.
+      </p>
+      <p>
+        If you cannot reach an event organizer or forum administrator and/or
```

Comment:
If you cannot reach an event organizer or forum administrator and/or it is an emergency, please call 911 and/or remove yourself from the situation.